src/lib/seam/components/AccessCodeTable/AccessCodeTable.tsx

```diff
@@ -229,6 +227,22 @@ export function AccessCodeTable({
           disableDeleteAccessCode={disableDeleteAccessCode}
         />
       </TableBody>
+
+      {isError && (
+        <Snackbar
```

Comment:
Should we build the conditional render logic into the `Snackbar` component since otherwise it it always going to be `isError && <...>`

Cleaner if we do

```
<Snackbar
  visible={isError}
```

Reply:
#510